tests/doc/tst_doc_images.py

```diff
@@ -185,31 +185,35 @@ def _test_both_images_exist(filename, docs_image_path, cached_image_path):
 
 
 def _test_compare_images(test_name, docs_image_path, cached_image_path):
-    docs_image = pv.read(docs_image_path)
-    cached_image = pv.read(cached_image_path)
-
-    # Check if test should fail or warn
-    error = pv.compare_images(docs_image, cached_image)
-    fail_msg = _check_compare_fail(test_name, error)
-    warn_msg = _check_compare_warn(test_name, error)
-    if fail_msg:
-        # Check if test case is flaky test
-        if test_name in FLAKY_TEST_CASES:
-            # Compare build image to other known valid versions
-            success_path = _is_false_positive(test_name, docs_image)
-            if success_path:
-                # Convert failure into a warning
-                warn_msg = fail_msg + (
-                    '\nTHIS IS A FLAKY TEST. It initially failed (as above) but passed when '
-                    f'compared to:\n\t{success_path}'
-                )
-                fail_msg = None
-            else:
-                # Test still fails
-                fail_msg += (
-                    '\nTHIS IS A FLAKY TEST. It initially failed (as above) and failed again for '
-                    f'all images in \n\t{Path(FLAKY_IMAGE_DIR, test_name)!s}.'
-                )
+    try:
+        docs_image = pv.read(docs_image_path)
+        cached_image = pv.read(cached_image_path)
+
+        # Check if test should fail or warn
+        error = pv.compare_images(docs_image, cached_image)
```

Comment:
Assuming we don't want to necessarily catch this error in other locations, it would be more precise to only wrap the call to `pv.compare_images` in the `try` block, keep the `except` as is and then put the rest of this in an `else:` block.

Reply:
There is another call to `compare_images`  later on in `_is_false_positive`.  So, we need to wrap both calls somehow